File: POO/PHP/Car.php
<?php
require_once('Account.php');
class Car
{
    public $id;
    public $lisence;
    public $driver;
    public $passengenger;

    public function __construct($lisence, $driver)
    {
        $this->lisence = $lisence;
        $this->driver = $driver;
    }

    public function __printDataCar()
    {
        echo 'Licencia: ' + $this->lisence + '<br/>';
        echo 'Conductor: ' + $this->driver->name + '<br/>';
    }
}
?>

File: POO/Java/Car.java
package Java;

public class Car 
 {
     Integer id;
     String lincense;
     Account driver;
     private Integer passegennger;

     public Car(String license, final Account driver)
     {
         this.lincense = license;
         this.driver = driver;

     }
    
    void printDataCar()
    {
        System.out.println("Placa: " + lincense + ", Name Driver: " + driver.name + "Pasajeros: " + passegennger);
    }
    public Integer getPassenger()
    {
        return passegennger;

    }
    public void setPassenger(Integer passenger)
    {
        if ( passenger == 4)
        {
            this.passegennger = passenger;
        }
        else
        {
            System.out.println("Nesesitas asignar 4 asientos");
        }
       
        
 
    }
 }